libs/web/src/components/common/Footer/Footer.tsx

```diff
@@ -144,7 +144,7 @@ const Footer = ({isHomePage = false}: FooterProps) => {
           </div>
           <div className={styles.copyright}>
             <p className={clsx("desktopOnly", "mc-type-m")}>
-              &copy; {currentYear} Mira Finance
+              &copy; {currentYear} Microchain
```

Comment:
Here do we need finance?
```suggestion
              &copy; {currentYear} Microchain Finance
```